kubernetes/package-kafka/README.md

```diff
@@ -0,0 +1,53 @@
+# Deploy kafka package to Apache OpenWhisk
+
+This project is to deploy kafka package to local Apache OpenWhisk on a K8s using YAML file.
+
+## Step 1. Install kafka provider
+Config the environment with correct values in yml file, and then
+```
+kubectl apply -f kafkaprovider.yml
+```
+### Mandatory Environment Variables
```

Comment:
Most of these are already define in config maps.  Get them from there in the yaml file and create a packages.kafkaprovider configmap for any that are specific to this package.